app/src/main/java/com/auth0/android/lock/app/DemoActivity.java

```diff
@@ -136,8 +136,8 @@ public void onClick(View view) {
             }
         });
 
-        Button btnShowHostedLoginPage = (Button) findViewById(R.id.btn_show_hosted_login_age);
-        btnShowHostedLoginPage.setOnClickListener(new View.OnClickListener() {
+        Button btnShowUniversalLogin = (Button) findViewById(R.id.btn_show_universal_login);
```

Comment:
tbh Would just call these WebAuth or Login

Reply:
it's just a button in the sample app. I don't mind how we call it. Its your call 😛